bundle/cert-manager-operator/1.1.0/certmanagers.operator.cert-manager.io.crd.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: certmanagers.operator.cert-manager.io
+spec:
+  group: operator.cert-manager.io
+  names:
+    kind: CertManager
+    listKind: CertManagerList
+    plural: certmanagers
+    singular: certmanager
+  scope: Namespaced
+  subresources:
+    status: {}
+  validation:
+    openAPIV3Schema:
+      type: object
+      x-kubernetes-preserve-unknown-fields: true
```

Comment:
Is this still needed?

Reply:
Probably not, openshift is already on k8s 1.19